### ISSUE TITLE
It is not possible to hide temporarily disabled entities

### ISSUE THREAD

**Issue by vitjurasek** [thomasloven/lovelace-auto-entities#429]
In the list of sorted socket consumption entities, I also have entities that are deliberately disabled during the winter. 

Unfortunately, the list lists it as "Entity unavailable: sensor.shelly_power". 

Would it be possible to add a filter to ignore the disabled entities?

**Comment by rugbyopera** [comment]
```filter:
  include:
    [whatever]
  exclude:
    hidden-by: '*'

**Comment by miguelarios** [comment]
> ```
> filter:
>   include:
>     [whatever]
>   exclude:
>     hidden-by: '*'
> ```

This doesn't work for me. Small typo its with a `_` not `-` in `hidden_by`.

**Comment by bcutter** [comment]
`hidden_by: '*'` is not working. What's an appropriate solution for this demand?

`restored: true` comes close, but unfortunately also filters enabled entities with that attribute (= false positives).

```
filter:
  exclude:
    - attributes:
        restored: true
```

Probably auto-entities will never be able to filter disabled entities, as that information simply seems to be unavailable in the state machine auto-entities relies on.

As a workaround (and this is pretty annoying most of the time as it creates constant manual work), I apply a label to disabled and "undesired" (to be filtered out) entities for now, then excluding entities with that label.

```
filter:
  exclude:
    - label: Disabled_Manually
```

**Comment by dcapslock** [comment]
hidden_by: '*' is for when the Visible toggle is used.

It may be possible to have a `disabled:` filter. If desred suggest to put in a FR on the [LFT Fork](https://github.com/Lint-Free-Technology/lovelace-auto-entities)

EDIT: Yep, `disabled_by` is a separate property and as auto-entities deals with config entities for these properties, and not state machine, it should be a straightforward implmentation.

**Comment by bcutter** [comment]
Interesting. You did fork AU? The original repo here is still maintained, isn't it? Why should one migrate to the fork?

**Comment by dcapslock** [comment]
Yes. No release in nearly nine months. A bunch of PRs of mine were merged but not released so quite confusing (e.g. readme notes state_translated but its not in any released build, and still an issue if you use 1.16.1). I had another half dozen PRs waiting, all to do with important aspects like cache, clearing cache when config on server changes etc. So that is why.

**Comment by dcapslock** [comment]
@bcutter checking on this. auto-entities current does not include any entities that don't have a state, so if you Disable an entity it will not show at all as it does not have a state. So can I check how you are disabling your entities? Is in a special option in an integration? Are you using auto-entities in a custom card.

While I could add `disabled_by` my testing show it will do nothing for the scenario of disbling an entity in Home Assistant

**Comment by bcutter** [comment]
Quite simple:
1. I disable them via the regular way (entity option in UI)
2. In include I use `state: unavailable`, which also applies for disabled entities. Now I want to filter explicitly for the disabled ones, because there are several other reasons entities can have that unavailable state.

**Comment by dcapslock** [comment]
@bcutter what I find is that disabled entities will stay in states for a short while, but once they are not in states, they won't show in auto-entities, as it has a final check on whether the entitiy is in states. They is why there is no specific flters around disabled, as they will never show. The best way to check is to restart Home Assistant after disabling or enabling entities.

Now if you want to be able to show what entities are disabled in auto-entities, this could be an option to propose as a feature request on the LFT fork, which is where I am now doing my contributions.

**Comment by mmerickel** [comment]
For me I was seeing entities forever after disabling an integration. Restarting fixed it. Thank you for the clues.
